d3a_api_client/__init__.py

```diff
@@ -1,5 +1,8 @@
 from abc import ABC, abstractmethod
 
+# *IMPORTANT*: Don't manually change the version here. Use the 'bumpversion' utility.
+__version__ = '0.11.0'
```

Comment:
This was also added here because there is no way to import the one in the setup.py, I'll update our documentation bump command to also bump this one

Reply:
Just a question here. Is bumbversion able to change the version here, too? I thought that it only looked into setup.py

Reply:
@hannesdiedrich Yes I pointed the setup.cfg to it